superset/extensions/ssh.py

```diff
@@ -34,6 +34,7 @@ class SSHManager:
     def __init__(self, app: Flask) -> None:
         super().__init__()
         self.local_bind_address = app.config["SSH_TUNNEL_LOCAL_BIND_ADDRESS"]
+        TUNNEL_TIMEOUT = app.config["SSH_TUNNEL_TIMEOUT"]
```

Comment:
does this set the `TUNNEL_TIMEOUT` constant back to sshtunel? or we still need to invoke sshtunel. TUNNEL_TIMEOUT = "value"
can we add a test for this?